Monika After Story/game/definitions.rpy

```diff
@@ -19,6 +19,15 @@ python early:
     import traceback
     _dev_tb_list = []
 
+    def dummy():
```

Comment:
Thinking we can do this to make it more general use to just about any override func and replace with the dummy func w/o needing to make another one if it takes args

```suggestion
    def dummy(*args, **kwargs):
```